src/olympia/reviewers/utils.py

```diff
@@ -255,12 +256,24 @@ def render_addon_name(self, record):
         ).count()
         if listed_versions:
             url = reverse('reviewers.review', args=[record.slug])
-            rval.append(
-                '<a href="%s">%s</a>' % (
-                    url,
-                    self.listed_text.format(listed_versions)
+            # The MAD queue filters on the current_version of listed add-ons,
+            # so there can only be 0 or 1 listed versions per add-on in the
+            # queue. However, the filter above does not filter on that, so
+            # we just hide the count for listed entries.
+            if self.single_listed_version:
+                rval.append(
+                    '<a href="%s">%s</a>' % (
+                        url,
+                        self.listed_text.format(listed_versions)
+                    )
+                )
+            else:
+                rval.append(
+                    '<a href="%s">%s</a>' % (
```

Comment:
I'm confused, and maybe I'm just blind, but this statement looks identical to the one above it, in the other path of the `if`.

Reply:
You are right. I initially added this because if `single_listed_version` is true, we don't have nothing to format `self.listed_text`, but it seems `format` is just ignored if there is no placeholder in the string.

Reply:
.a2388487cc